src/python/pants/backend/codegen/protobuf/go/rules.py

```diff
@@ -579,24 +579,26 @@ async def setup_go_protoc_plugin() -> _SetupGoProtocPlugin:
         ),
     )
 
-    download_sources_result = await Get(
+    await Get(
```

Comment:
Maybe add a comment that this process is being executed for its side effect of ensuring the cache contains the expected sources?